include/circt/Dialect/FIRRTL/FIRRTLExpressions.td

```diff
@@ -610,4 +610,17 @@ def HWStructCastOp : FIRRTLOp<"hwStructCast", [NoSideEffect]> {
     "$input attr-dict `:` functional-type($input, $result)";
 
   let verifier = "return ::verifyHWStructCastOp(*this);";
-}
\ No newline at end of file
+}
+
+def BitCastOp: FIRRTLOp<"bitcast", [NoSideEffect]> {
+  let summary = [{
+    Reinterpret one value to another value of the same size and
+    potentially different type. This op is lowered to hw::BitCastOp.
```

Comment:
Instead of "This op is lowered to hw::BitCastOp.", how about "This op follows the same endianness policy as `hw::BitCastOp`."